doc/techref/justification_codes.md

```diff
@@ -0,0 +1,120 @@
+---
+file_format: mystnb
+---
+
+# Justification codes
+
+To place plot embellishments, such as scalebars, directional roses, colorbars,
+legends, text, and images on a figure two points have to be specified: a point
```

Comment:
Here, we could link to the reference pages for reference and anchor points which will be added in PR #4052 .